core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -479,7 +479,10 @@ protected Node visitShowCreate(ShowCreate node, Void context)
 
                 accessControl.checkCanShowCreateTable(session.toSecurityContext(), new QualifiedObjectName(catalogName.getValue(), schemaName.getValue(), tableName.getValue()));
 
-                String sql = formatSql(new CreateView(QualifiedName.of(ImmutableList.of(catalogName, schemaName, tableName)), query, false, viewDefinition.get().getComment(), Optional.empty())).trim();
+                CreateView.Security security = viewDefinition.get().isRunAsInvoker()
+                        ? CreateView.Security.INVOKER
```

Comment:
Static import ?? Sorry missed it before.

Reply:
Done (not a fan of statically importing fields, but a style guide is a style guide!)